src/main/java/org/codehaus/groovy/vmplugin/VMPlugin.java

```diff
@@ -179,4 +181,17 @@ static String getJavaVersion() {
     default Map<String, Set<String>> getDefaultImportClasses(String[] packageNames) {
         return Collections.emptyMap();
     }
+
+    /**
+     * Returns the list of record component names or the empty list if
+     * the class is not a record or running on a pre16 JDK.
+     *
+     * @param maybeRecord the class in question
+     * @return the default list of names
+     * @since 4.0.14
```

Comment:
4.0.14 was released, so the version should be 4.0.15?

Reply:
Indeed, fixed.